kernel-default.properties

```diff
@@ -458,69 +452,69 @@ mosip.role.admin.masterdata.getbiometrictypes=ZONAL_ADMIN,PRE_REGISTRATION_ADMIN
 mosip.role.admin.masterdata.getbiometrictypeslangcode=ZONAL_ADMIN,GLOBAL_ADMIN,PRE_REGISTRATION_ADMIN,REGISTRATION_SUPERVISOR,REGISTRATION_PROCESSOR,REGISTRATION_OFFICER
 mosip.role.admin.masterdata.getbiometrictypescodelangcode=ZONAL_ADMIN,PRE_REGISTRATION_ADMIN,GLOBAL_ADMIN,REGISTRATION_SUPERVISOR,REGISTRATION_PROCESSOR,REGISTRATION_OFFICER
 mosip.role.admin.masterdata.postbiometrictypes=ZONAL_ADMIN,GLOBAL_ADMIN
-mosip.role.admin.masterdata.postblocklistedwordswords=ZONAL_ADMIN,GLOBAL_ADMIN,MASTERDATA_ADMIN
-mosip.role.admin.masterdata.postblocklistedwords=GLOBAL_ADMIN,ZONAL_ADMIN,MASTERDATA_ADMIN
-mosip.role.admin.masterdata.putblocklistedwords=GLOBAL_ADMIN,ZONAL_ADMIN,MASTERDATA_ADMIN
-mosip.role.admin.masterdata.putblocklistedwordsdetails=GLOBAL_ADMIN,ZONAL_ADMIN,MASTERDATA_ADMIN
+mosip.role.admin.masterdata.postblocklistedwordswords=ZONAL_ADMIN,GLOBAL_ADMIN
+mosip.role.admin.masterdata.postblocklistedwords=GLOBAL_ADMIN,ZONAL_ADMIN
+mosip.role.admin.masterdata.putblocklistedwords=GLOBAL_ADMIN,ZONAL_ADMIN
+mosip.role.admin.masterdata.putblocklistedwordsdetails=GLOBAL_ADMIN,ZONAL_ADMIN
 mosip.role.admin.masterdata.deleteblocklistedwordsword=GLOBAL_ADMIN,ZONAL_ADMIN
-mosip.role.admin.masterdata.getblocklistedwordsall=GLOBAL_ADMIN,PRE_REGISTRATION_ADMIN,ZONAL_ADMIN,MASTERDATA_ADMIN
-mosip.role.admin.masterdata.postblocklistedwordssearch=GLOBAL_ADMIN,ZONAL_ADMIN,MASTERDATA_ADMIN
-mosip.role.admin.masterdata.postblocklistedwordsfiltervalues=GLOBAL_ADMIN,ZONAL_ADMIN,MASTERDATA_ADMIN
-mosip.role.admin.masterdata.patchblocklistedwords=GLOBAL_ADMIN,ZONAL_ADMIN,REGISTRATION_ADMIN,MASTERDATA_ADMIN
+mosip.role.admin.masterdata.getblocklistedwordsall=GLOBAL_ADMIN,PRE_REGISTRATION_ADMIN,ZONAL_ADMIN
+mosip.role.admin.masterdata.postblocklistedwordssearch=GLOBAL_ADMIN,ZONAL_ADMIN
+mosip.role.admin.masterdata.postblocklistedwordsfiltervalues=GLOBAL_ADMIN,ZONAL_ADMIN
+mosip.role.admin.masterdata.patchblocklistedwords=GLOBAL_ADMIN,ZONAL_ADMIN,REGISTRATION_ADMIN
 mosip.role.admin.masterdata.getdeviceslanguagecode=ZONAL_ADMIN,GLOBAL_ADMIN,PRE_REGISTRATION_ADMIN,REGISTRATION_PROCESSOR,REGISTRATION_SUPERVISOR,REGISTRATION_OFFICER
 mosip.role.admin.masterdata.getdeviceslanguagecodedevicetype=ZONAL_ADMIN,PRE_REGISTRATION_ADMIN,GLOBAL_ADMIN,REGISTRATION_PROCESSOR,REGISTRATION_SUPERVISOR,REGISTRATION_OFFICER
-mosip.role.admin.masterdata.postdevices=GLOBAL_ADMIN,ZONAL_ADMIN,MASTERDATA_ADMIN
```

Comment:
Can you pls cross check and confirm if these role changes are w.r.t role matrix published in our mosip docs

Reply:
@Varaniya @vishwa-vyom - We notice that MASTERDATA_ADMIN role is removed from the config, but as per role matrix it is required. can you pls confirm on this?
Ref: https://docs.mosip.io/1.2.0/modules/administration/admin-portal-user-guide#admin-roles-and-their-default-accessibility-matrix

Reply:
@dhanendra06 as discussed with you please check when this was removed? is this removed as part of LTS version?

Reply:
we have not found who has removed this role I have checked back to 2-year commits  so added it again because roles are there in develop and master

Reply:
Added MASTERDATA_ADMIN roles @Varaniya201 @vishwa-vyom

Reply:
Resolving this comment, since it is added by @dhanendra06, if any concerns, please reopen this comment